bisqapps/shared/presentation/src/commonMain/kotlin/network/bisq/mobile/presentation/ui/screens/SplashPresenter.kt

```diff
@@ -0,0 +1,30 @@
+package network.bisq.mobile.presentation.ui.screens
```

Comment:
For the "screens" package, I see the idea is to group presentation (presenter + ui) in a by use case manner.

I think this is great, to make it more explicit can we change the name of the `screens` subpackage to `uicases` or `features` and have subpackages for each grouping related UI and presenter ?

In cases where one presenter serves many UIs the presenter can be in a "presenters" package or in the root `uicases` folder

what do you think?

Reply:
This will be better. Thanks. I will also change the folder structure accordingly? `screens` -> `uicases`?

Reply:
that looks beeeeaaaauuuutiiifulll :D :D